### PR TITLE
Update sponsor/donation content

### PR DESCRIPTION
**Pull request by rgommers**
As discussed on the mailing list: https://mail.python.org/pipermail/scipy-dev/2019-April/023457.html

- remove "sponsored by Enthought" from banner
- update SciPy library donation page
- add NumFOCUS to front page as fiscal sponsor

This also needs https://github.com/scipy/scipy-sphinx-theme/pull/11. And docs.scipy.org needs an update as well. Will update that once the scipy-sphinx-theme PR is merged.

Note that this just changes the essentials. A larger overhaul is still desirable.

### CODE REVIEW

**Comment by rgommers** [comment]
As I mentioned on the mailing list: I plan to merge this in two days.

**Comment by ilayn** [comment]
Ah this is a perfect opportunity to mention the logo text kerning issue :) Well basically this https://www.xkcd.com/1015/

image

**Comment by rgommers** [comment]
> Do we still have the original? I can make a vectorized version of this fairly quickly.

I'm afraid not. At least I couldn't find anything but the tiny gif that I modified. I'll ask on the mailing list.

**Comment by rgommers** [comment]
Updated with the scipy-sphinx-theme change, and added a few sentences to the almost-empty front page.

Note that http://conference.scipy.org/ is a separate site which still shows "sponsored by Enthought", which is how it should be.

Time to get this up, in it goes.

@ilayn we can do the better logo in a separate PR.

**Comment by ilayn** [comment]
Sure no problem. I will also send something to the mailing list about it, when I have the chance, about switching to an open source font which is very close to the existing typeface.

**Comment by rgommers** [comment]
that sounds like a good idea

File: www/index.rst [hunk 1]
```diff
@@ -184,14 +184,26 @@ particular, these are some of the core packages:
   </div>
   </li>
 
+  <li class="span12">
+  <div class="thumbnail">
+  <div class="pull-left img">
+    <a href="http://pandas.pydata.org/">
```
Comment (WarrenWeckesser) [inline www/index.rst]: Ralf, this is a URL for Pandas.  Shouldn't this link to NumFOCUS?

Comment (rgommers) [reply www/index.rst]: oops. thanks for catching that. fixed in gh-293

**Pull request merged**